homeassistant/components/denonavr/media_player.py

```diff
@@ -73,99 +54,32 @@
     | SUPPORT_PLAY
 )
 
-DENON_ZONE_SCHEMA = vol.Schema(
-    {
-        vol.Required(CONF_ZONE): vol.In(CONF_VALID_ZONES, CONF_INVALID_ZONES_ERR),
-        vol.Optional(CONF_NAME): cv.string,
-    }
-)
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Optional(CONF_HOST): cv.string,
-        vol.Optional(CONF_NAME): cv.string,
-        vol.Optional(CONF_SHOW_ALL_SOURCES, default=DEFAULT_SHOW_SOURCES): cv.boolean,
-        vol.Optional(CONF_ZONES): vol.All(cv.ensure_list, [DENON_ZONE_SCHEMA]),
-        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
-    }
-)
 
-NewHost = namedtuple("NewHost", ["host", "name"])
-
-
-def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
If we would import the legacy yaml config to a config entry via the config flow we won't break existing user's options as much. Then we can wait with adding the options flow to the next PR without big impact.

Reply:
I don't know how to mix the Config Flow with the old implementation and probably am going to run in a whole bunch of problems with that. Not going to open that can of worms...
I do understand that this will be a breaking change but that is the case with many of the already accepted Config Flow implementations.

Reply:
not needed anymore